Cargo.toml

```diff
@@ -26,7 +26,7 @@ socks-proxy = ["socks"]
 [dependencies]
 base64 = "0.13"
 chunked_transfer = "1.2.0"
-cookie = { version = "0.14", features = ["percent-encode"], optional = true}
```

Comment:
This feature is no longer needed (but still enabled by `cookie_store`, see https://github.com/pfernie/cookie_store/issues/13)